example/components/slide/slide.vue

```diff
@@ -138,7 +138,7 @@
           child.style.width = slideWidth + 'px'
           width += slideWidth
         }
-        if (this.loop && !isResize) {
+        if (this.loop && !isResize && this.children.length > 1) {
```

Comment:
(this.loop && !isResize) {this.children.length > 1) {
```suggestion
        if (this.loop && !isResize && this.children.length > 1) {
```